src/vt-tv/render/render.cc

```diff
@@ -307,14 +307,32 @@ Render::createObjectMapping_(PhaseType phase) {
   return object_mapping;
 }
 
+template <typename T, typename U>
+void Render::addRankArray(
+  vtkNew<vtkPolyData>& pd_mesh, PhaseType phase, std::string const& key
+) {
+  vtkNew<U> array;
+  std::string array_name = key;
+  array->SetName(array_name.c_str());
+  array->SetNumberOfTuples(this->n_ranks_);
+
+  for (uint64_t rank_id = 0; rank_id < this->n_ranks_; rank_id++) {
+    auto const& cur_rank_info = info_.getRanks().at(rank_id);
+    auto const& value = info_.getRankUserDefined(cur_rank_info, phase, key);
+    //fmt::print("phase={}, key={}, rank_id={}\n", phase, key, rank_id);
+    array->SetTuple1(rank_id, std::get<T>(value));
+  }
+  pd_mesh->GetPointData()->AddArray(array);
+}
+
 vtkNew<vtkPolyData> Render::createRankMesh_(PhaseType phase) {
   fmt::print("\n\n");
   fmt::print("----- Creating rank mesh for phase {} -----\n", phase);
   vtkNew<vtkPoints> rank_points_;
   rank_points_->SetNumberOfPoints(this->n_ranks_);
 
   vtkNew<vtkDoubleArray> rank_arr;
-  std::string rank_array_name = "Rank " + this->rank_qoi_;
+  std::string rank_array_name = this->rank_qoi_;
```

Comment:
@lifflander This is the change that causes the failure.

To fix the pipelines, we just need to update the expected rank meshes in `tests/expected/default` and `tests/expected/ccm_example`.

@tlamonthezie Might have an idea of the best way to go about this.